.github/workflows/ci.yml

```diff
@@ -1,5 +1,7 @@
 name: CI
 
+
+
```

Comment:
why the empty lines?

Reply:
Just to trigger rebuild. Will drop this commit after pipeline succeed

Reply:
you may create an empty commit `git commit --allow-empty`

Reply:
Dropped this commit